src/date.py

```diff
@@ -26,20 +26,34 @@ def build(self, date_object):
         return datetime(date_object["year"], date_object["month"], date_object["day"],
                         date_object["hour"], date_object["minute"], date_object["second"])
 
-    def from_exif(self, exif, user_regex=None):
+    def from_exif(self, exif, timestamp=False, user_regex=None):
```

Comment:
Can we unify the default values of this method? I used `None` across the project. I guess it will be better to use it here and in the tests too.

Reply:
sure, it works the same way in if tests, np